.github/workflows/release.yml

```
@@ -1,6 +1,6 @@
 name: release
 
-on: [publish]
+on: [create]
```

On second thoughts, I can't tell if this works if you push a tag instead of creating it via releases? It might be worth doing
```suggestion
on:
  push:
    tags:
      - 'v*'
```